### DISCUSSION TITLE
Cannot login into the forum

### DISCUSSION THREAD

**Discussion by alex180500** [typst/typst#5153] [General]
![image](https://github.com/user-attachments/assets/c48e2603-6790-411d-9321-a9d2c9b57522)

What is going on?

**Comment by alex180500** [comment]
Seems like they are resetting login methods

**Reply by laurmaedje** [reply]
Plugins (including the authentication plugin) are currently not loaded correctly on the Discourse Forum. Unfortunately, we couldn’t yet figure out why.
